spec/back_end/features/list_spaces_spec.rb

```diff
@@ -1,6 +1,6 @@
 feature 'list spaces' do
   scenario 'an authenticated user can list a space' do
-    User.create({ email: 'abacon@test.com', password: 'pass1' })
+    create_user
```

Comment:
This won't work as the user needs to be created by the application in the browser and not hardcoded. We have implemented on the encryption branch.